honeybee_radiance_command/rpict.py

```diff
@@ -0,0 +1,146 @@
+"""rpict command."""
+
+from honeybee_radiance.view import View
```

Comment:
I missed this one - you can't have dependencies on honeybee_radiance. The View in this case is a view file not a View object.